lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -19,12 +19,13 @@
           </p>
         </div>
         <div ng-if="main.selection.hasImage()">
-          <div class="step-selection-text">
+          <div class="step-selection-text"
+            uib-popover="{{ main.selection.getImagePath() }}"
+            popover-title="{{ main.selection.getImagePath() | basename }} - {{ main.selection.getImageSize() | gigabyte | number:1 }}GB"
+            popover-trigger="{'mouseenter': 'none', 'outsideClick': 'outsideClick'}">
```

Comment:
Hm, really?

> Through the $uibTooltipProvider, you can change the way **tooltips and popovers** behave by default

That'd definitely be weird. I can think of two things:

- Check if they define a `$uibPopoverProvider` and the simply made a typo on the docs
- Check if we're running the latest version, and try upgrading if not